### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by hswopeams**
Update overview.md and graphics.
deployments.md wont' be updated until we release v1.0.0 to mainnet

### CODE REVIEW

**Review by cliffhall — DISMISSED** [review]
Looks good. I'll just say, though that the inheritance graph is absurd to include because like most generated graphs of non-trivial systems, it's unreadable. It should be done manually so that layout can be attended to.

**Comment by hswopeams** [comment]
> Looks good. I'll just say, though that the inheritance graph is absurd to include because like most generated graphs of non-trivial systems, it's unreadable. It should be done manually so that layout can be attended to.

It's indeed not ideal.

File: docs/contracts/overview.md [hunk 1]
```diff
@@ -15,13 +15,15 @@ are based on two NFT standards,
...
-* `ERC1155ERC721`: Token factory  
+* `VoucherSets`: VoucherSets token contract. Conforms to the ERC1155 standard.
+* `Vouchers`: Vouchers token contract. Conforms to the ERC721 standard
 * `TokenRegistry`: Restrictions on the allowed escrowed amounts and registry of token wrappers
 * `VoucherKernel`: Main business logic  
 * `UsingHelpers`: Common utilities
 * `DAITokenWrapper`: Provides a uniform interface for calling the `permit `function on the DAI token
 * `ERC1155NonTransferable`: Non-transferrable token, the possession of which allows a buyer to commit to an item following a special quest or campaign.
 * `Gate`: Allows users of the protocol to gate access to a given VoucherSet by requiring that any address attempting to commit to an item in the given VoucherSet have a given ERC1155 before they access.
+* `MetaTransactionReceiver`: Base contract that allows the ERC1155NonTransferable contract to receive and proces meta transactions
```
Comment (bidyut-arianelabs) [inline docs/contracts/overview.md:26]: typo in `process`.

**Review by zajck — COMMENTED** [review]
Is the Happy Path section really needed? Similar stuff is written already in "Process"

Also `BosonRouter.requestCreateOrder()` and `BosonRouter.requestVoucher()` are not exact functions that exist in the contract so it might be confusing for someone reading it the first time.

File: docs/contracts/overview.md [hunk 2]
```diff
@@ -187,25 +193,24 @@ its lifecycle (defined in
...
+periods being triggered.  
 
 ### Services in the background
   
-A scheduled process is running in the backend that flags the vouchers when 
-redemption was made and when wait periods expire. Anybody could be executing 
-these functions, marked as external, the backend is currently running them for 
-convenience: `VoucherKernel.triggerExpiration()`, 
+There are a few functions that can be called by an application built on Boson Protocol to mark vouchers as 
+expired or finalized or to wtidhraw funds. These functions can be called on a scheduled basis or as
```
Comment (bidyut-arianelabs) [inline docs/contracts/overview.md:212]: typo `wtidhraw`

**Review by bidyut-arianelabs — CHANGES_REQUESTED** [review]

File: docs/contracts/overview.md (hunk 1 above)
Comment (hswopeams) [reply docs/contracts/overview.md:26]: It's the correct spelling in Dutch. ;-)

Comment (hswopeams) [reply docs/contracts/overview.md]: Fixed.

File: docs/contracts/overview.md (hunk 2 above)
Comment (hswopeams) [reply docs/contracts/overview.md]: Fixed.

**Review by bidyut-arianelabs — APPROVED** [review]
Looks good to me 👍

**Review by cliffhall — APPROVED** [review]
LGTM! 👍

**Pull request merged**